docs/manual-EN/5.appendix/sql-ngql.md

```diff
@@ -0,0 +1,194 @@
+
+<return_list>
+    <col_name> [AS <col_alias>] [, <col_name> [AS <col_alias>] ...]
+```
+
+### Data Control Language (DCL)
+
+DCL includes commands such as `GRANT` and `REVOKE` which mainly deals with the rights, permissions and other controls of the database system.
+
+Name                    | SQL                   | nGQL
+-------------------------| ------------------------ | -----------
+Create user | CREATE USER | CREATE USER
+Drop user | DROP USER | DROP USER
+Change password | SET PASSWORD | CHANGE PASSWORD
+Grant privileges | GRANT `<priv_type>` ON [object_type] TO `<user>`| GRANT ROLE `<role_type>` ON `<space>` TO `<user>`
+Revoke privileges | REVOKE `<priv_type>` ON [object_type] TO `<user>` | REVOKE ROLE `<role_type>` ON `<space>` FROM `<user>`
+
+## CRUD
+
+This section gives some query examples for your reference.
+
+### Inserting Data
+
+```sql
+mysql> INSERT INTO player VALUES (100, 'Tim Duncan', 42);
+
+nebula> INSERT VERTEX player(name, age) VALUES 100: ('Tim Duncan', 42);
+```
+
+### Querying Data
+
+Find the player in the `player` table whose id is 100 and output the `name` property in RDBMS:
+
+```sql
+mysql> SELECT player.name FROM player WHERE player.id = 100;
+
+nebula> FETCH PROP ON player 100 YIELD player.name;
+```
+
+### Updating Data
+
+```sql
+mysql> UPDATE player SET name = 'Tim';
+
+nebula> UPDATE VERTEX 100 SET player.name = "Tim";
+```
+
+### Deleting Data
+
+```sql
+mysql> DELETE FROM player WHERE name = 'Tim';
+
+nebula> DELETE VERTEX 121;
+nebula> DELETE EDGE follow 100 -> 200;
+```
+
+## Sample Queries
+
+### Query 1
+
+Find players who are younger than 36.
+
+```sql
+mysql> SELECT player.name
+FROM player
+WHERE player.age < 36;
+```
+
+The query in nGQL is a bit different because we need to create an index before filtering a property. See [Index Doc](https://docs.nebula-graph.io/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/) for more information.
+
+```ngql
+nebula> CREATE TAG INDEX player_age ON player(age);
+nebula> REBUILD TAG INDEX player_age OFFLINE;
+nebula> LOOKUP ON player WHERE player.age < 36;
+```
+
+### Query 2
+
+List Tim Duncan and the team he served.
+
+```sql
+mysql> SELECT a.id, a.name, c.name
+FROM player a
+JOIN serve b ON a.id=b.player_id
+JOIN team c ON c.id=b.team_id
+WHERE a.name = 'Tim Duncan';
+```
+
+```ngql
+nebula> CREATE TAG INDEX player_name ON player(name);
+nebula> REBUILD TAG INDEX player_name OFFLINE;
+nebula> LOOKUP ON player WHERE player.name == 'Tim Duncan' YIELD player.name AS name | GO FROM $-.VertexID OVER serve YIELD $-.name, $$.team.name;
+```
+
+### Query 3
+
+Search Tim Duncan's player mates.
+
+```sql
+mysql> SELECT a.id, a.name, c.name
+FROM player a
+JOIN serve b ON a.id=b.player_id
+JOIN team c ON c.id=b.team_id
+WHERE c.name IN (SELECT c.name
+FROM player a
+JOIN serve b ON a.id=b.player_id
+JOIN team c ON c.id=b.team_id
+WHERE a.name = 'Tim Duncan');
+```
+
+In nGQL we use pipe as the input of the second clause in this case.
```

Comment:
suggestion: "In nGQL, pipes are used to pass the output of the previous statement as the input for the next statement." ?